spec/requests/device_story_comments_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe DeviceStoryCommentsController, type: :request do
```

Comment:
> Any idea why I get the NoMethodError in rspec when I use sign_in user?

```suggestion
RSpec.describe DeviceStoryCommentsController, type: :controller do
```

@Nakushi64 If changed like above, it should solve error.

`Devise::Test::ControllerHelpers` that defines `sigin_user` method will be included. It is defined in `spec/spec_helper.rb` (`config.include Devise::Test::ControllerHelpers, type: :controller`)